cvat-core/src/annotations.js

```diff
@@ -284,6 +284,22 @@
         return result;
     }
 
+    async function importDataset(instance, format, file, updateStatusCallback) {
+        if (!(format instanceof String || typeof format === 'string')) {
+            throw new ArgumentError('Format must be a string');
+        }
+        if (!(instance instanceof Project)) {
+            throw new ArgumentError('Instance should ne a Project isntance');
+        }
+        if (!(typeof updateStatusCallback === 'function' || updateStatusCallback === null)) {
```

Comment:
You can set a default argument as an empty function: ``() => void`` and left one check only